### PR TITLE
Use HTML entities to prevent Liquid from processing this info

### PR DESCRIPTION
**Pull request by parkr**

### CODE REVIEW

**Comment by ashmaroli** [comment]
Guys, IMO the cleaner approach would be to use Liquid's `raw` tags instead. The primary reason being the massive hit that readability via GitHub interface took:
github com_jekyll_github-metadata_blob_c35f1192f8d410fb033a90c0ed37248e7efd2275_docs_site github md

**Comment by DirtyF** [comment]
@ashmaroli I believe you have to insert HTML comments to prevent `raw` tags for showing up on GitHub.

File: docs/site.github.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ Simply, reference any of the below keys as you would any other site configuratio
 For example, to list a project's name, you might write something like `The project is called {{ site.github.project_title }}` or to list an organization's open source repositories, you might use the following:
 
 ```liquid
-{% for repository in site.github.public_repositories %}
-  * [{{ repository.name }}]({{ repository.html_url }})
-{% endfor %}
```
Comment (DirtyF) [inline docs/site.github.md]:
```liquid
<!-- {% raw %} -->

{% for repository in site.github.public_repositories %}
  * [{{ repository.name }}]({{ repository.html_url }})
{% endfor %}

<!-- {% endraw %} -->

Comment (parkr) [reply docs/site.github.md]: Oooooo now there's a great idea! That's what I was looking for, @DirtyF!

**Pull request merged**